.github/workflows/setup.yml

```diff
@@ -6,6 +6,9 @@ on:
 permissions:
   contents: read
 
+env:
+  COSIGN_PASSWORD: 6EHvWD1BUk0yWdvm-GGNxA==
```

Comment:
confirmed with Miro this is for internal testing usage only, not part of a public action